### ISSUE TITLE
minor warning detail on nuget restore option

### ISSUE THREAD

**Issue by toddbellMSFT** [MicrosoftDocs/azure-devops-docs#9785]
TL;DR -  It may be worth considering appending to the parameter restoreNugetPackages' Optional comment, that it is deprecated and suggest the developer should use the restore nuget packages task [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/package/nuget?view=azure-devops#:~:text=Arguments%20%20%20%20Argument%20%20%20,the%20generat%20...%20%2020%20more%20rows%20) instead of solely relying on this parameter.

The details:
Nuget restore is only an option if the developer has installed the tool previously in their pipeline:

##[warning]The 'Restore NuGet Packages' option is deprecated. To restore NuGet packages in your build, add a NuGet Tool Installer task to your build definition.

I tried adding the parameter mentioned to my VSBuild task, but this above warning was un-expected. 

Nuget is already available to the azure pipeline vms, it's weird that I have to add the NuGetTool Installer task. I already needed to have a nuget restore on the project called first, I may as well call that task instead of trying this parameter.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 29f86c97-f534-50ad-bfb1-a2cf78c7af94
* Version Independent ID: acea40e3-512b-4d9a-9389-71b8b24327be
* Content: [Visual Studio Build build and release task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/visual-studio-build?view=azure-devops)
* Content Source: [docs/pipelines/tasks/build/visual-studio-build.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/tasks/build/visual-studio-build.md)
* Product: **devops**
* Technology: **devops-cicd-tasks**
* GitHub Login: @vijayma
* Microsoft Alias: **vijayma**
